### PR TITLE
Remove library() calls in shiny app

### LINKED ISSUE #111: Clean up library() calls
Per @xni7 notes [here](https://github.com/ASA-DIA-InteractiveSafetyGraphics/safetyGraphics/pull/104#discussion_r245345325).

### PR DESCRIPTION
**Pull request by jwildfire**
Trying to resolve #111, so dropped the various `library()` calls in the `global.R` and `UI.R` in the app. Now it's bombing though ... maybe namespace or description is off somehow? Or maybe we actually do need the library calls? 

```
install_github("ASA-DIA-InteractiveSafetyGraphics/safetyGraphics", ref = "no-library")
library(safetyGraphics)
safetyGraphicsApp()
```

@xni7 @pburnsdata @bzkrouse Thoughts?

### CODE REVIEW

**Comment by jwildfire** [comment]
(you can ignore the first commit here, just snuck in from #177)

File: inst/eDISH_app/global.R [hunk 1]
```diff
@@ -1,12 +1,3 @@
-# temporarily load deps
-library(safetyGraphics)
```
Comment (pburnsdata) [inline inst/eDISH_app/global.R]: I think you would keep the library() calls but drop the source() calls?

File: inst/eDISH_app/ui.R [hunk 2]
```diff
@@ -1,6 +1,3 @@
-library(shiny)
```
[the remark this answers is not in the document]
Comment (pburnsdata) [reply inst/eDISH_app/ui.R]: although maybe you need library(shiny)... idk

[the remark this answers is not in the document]
Comment (bzkrouse) [reply inst/eDISH_app/ui.R]: Oh yes, good point! We should put all lib calls in global.R only

**Comment by jwildfire** [comment]
This all makes sense to me! 

 @pburnsdata @bzkrouse I'm heading out shortly, but would one of you be willing to make the needed changes and get this merged to the dev branch?

**Pull request merged**